src/ARCtrl/Templates/Templates.Json.fs

```diff
@@ -0,0 +1,95 @@
+﻿module ARCtrl.Templates.Json
+
+open ARCtrl.Templates
+open ARCtrl.ISA
+open System
+#if FABLE_COMPILER
+open Thoth.Json
+#else
+open Thoth.Json.Net
+#endif
+
+module Organisation =
+    let encode = Encode.Auto.generateEncoder<Organisation>()
+
+    let decode = Decode.Auto.generateDecoder<Organisation>()
+
+module CompositeCell =
+
+    let encode = Encode.Auto.generateEncoder<CompositeCell>()
+
+    let decode = Decode.Auto.generateDecoder<CompositeCell>()
+
+module CompositeHeader =
+
+    let encode = Encode.Auto.generateEncoder<CompositeHeader>()
+
+    let decode = Decode.Auto.generateDecoder<CompositeHeader>()
+
+module ArcTable =
```

Comment:
Good to have! And I think you cood move the base-type converters to ISA.Json. 

But should we save the templates as `ISA-json`?

Reply:
I do not have any hard feelings on where to put the template logic. We can move it to ISA and ISA.Json.

Reply:
Ah sry a bit of miscommunication. I was referring to two things:
- Move the more general parsers (CompositeCell, ArcTable, ArcAssay... ) to ISA.Json
- Then the question whether we should store the templates as objects specified in [isa-json](https://isa-specs.readthedocs.io/en/latest/isajson.html), i.e. as [`processes`](https://isa-specs.readthedocs.io/en/latest/isajson.html#process-schema-json)